test/Microsoft.AspNetCore.WebSockets.ConformanceTest/AutobahnTests.cs

```diff
@@ -44,28 +48,31 @@ public AutobahnTests(ITestOutputHelper output) : base(output)
                     .ExcludeCase("9.*", "12.*", "13.*");
 
                 var cts = new CancellationTokenSource();
-                cts.CancelAfter(TimeSpan.FromMinutes(5)); // These tests generally complete in just over 1 minute.
+                cts.CancelAfter(TestTimeout); // These tests generally complete in just over 1 minute.
 
-                AutobahnResult result;
-                using (var tester = new AutobahnTester(loggerFactory, spec))
+                using (cts.Token.Register(() => logger.LogError("Test run is taking longer than expected duration of {timeoutMinutes:0.00} minutes. Aborting...", TestTimeout.TotalMinutes)))
```

Comment:
You say above that the expected duration is 1 minute, so `Test run is taking longer than maximum duration of...`.